### ISSUE TITLE
sawg: saturation of summing junction involving offset

### ISSUE THREAD

**Issue by jbqubit** [m-labs/artiq#764]
```$ artiq_corelog
[           7us]  INFO(runtime): ARTIQ runtime starting...
[        3855us]  INFO(runtime): software version 3.0.dev+1173.g07f5e991
[       10203us]  INFO(runtime): gateware version 3.0.dev+1173.g07f5e991
```

```
from artiq.experiment import *

class SAWGTest(EnvExperiment):
    """
    purpose: test saturation of summing junction involving offset (just before DAC output)
    test: sawg0 is reference tone; clamp amplitude of sawg1
    setup: scope run control single, manual trigger; 500ns/div, 100mV/div
    expectation:
        a0=1 is 120 mV amplitude
        if max=0.5 then sawg1 should not exceed 60 mV
        if min=-0.5 then sawg1 should not exceed -60 mV
    observation:
        maximum is violated
        minimum is not sharply clamped at -0.5
    """
    def build(self):
        print(self.__doc__)
        self.setattr_device("core")
        self.setattr_device("led")
        self.setattr_device("ttl_sma")

        self.setattr_device("sawg0")
        self.setattr_device("sawg1")

    @kernel
    def run(self):
        self.core.reset()
        delay(300*us)
        self.sawg0.reset()
        self.sawg1.reset()
        delay(10 * ms)
        self.test()

    @kernel
    def test(self):
        f0 = 2*MHz
        t0 = 1*us
        a0 = 0.5

        # starting condition
        self.sawg0.amplitude1.set(2*a0)
        self.sawg0.frequency1.set(f0)
        self.sawg1.amplitude1.set(a0)
        self.sawg1.frequency1.set(f0)
        self.sawg1.offset.set(a0)


        # set clipping amplitude
        self.sawg1.config.set_out_max(0.5)
        self.sawg1.config.set_out_min(-0.5)
```
![scope_13](https://user-images.githubusercontent.com/20543958/27653897-84aeef0c-5c0e-11e7-8569-aa5f1905115f.png)

**Comment by jordens** [comment]
Please consider closing this if understanding that this is AC coupled explains your observation.

**Comment by jordens** [comment]
ping

**Comment by jbqubit** [comment]
Notwithstanding #765, resolved as discussed in #762.
